app/views/static_pages/sponsors.html.haml

```diff
@@ -1,18 +1,20 @@
-.col-xs-8.col-xs-offset-2
-  %h1= t('.title')
+.sponsors
+  .col-md-10.col-md-offset-1
+    %h1= t('.title')
 
-  %hr
+    %hr
 
-  %h3= t('sponsors.thanks')
+    %p= t('sponsors.thanks')
 
-  - @sponsors.each do |sponsor|
-    .row.sponsor-row
-      = link_to t("sponsors.#{sponsor}.link"), target: '_blank' do
-        .col-xs-5
-          %div{class: t("sponsors.#{sponsor}.class_name")}
-            %span.l-text-hidden
-              = t("sponsors.#{sponsor}.name")
+    - @sponsors.each do |sponsor|
+      .row.sponsor-row
+        .col-sm-12
+        = link_to t("sponsors.#{sponsor}.link"), target: '_blank' do
+          .col-xs-5
+            %div{class: t("sponsors.#{sponsor}.class_name") + ' sponsor-logo'}
```

Comment:
let do `%div{class: "#{t("sponsors.#{sponsor}.class_name")} sponsor-logo"}` instead. Concatenating string is preferred in Ruby since it doesn't allocate an extra string. :smile_cat: